README.md

```diff
@@ -353,29 +353,42 @@ The steps below should allow you to build and run Notero yourself.
     has a directory named `extensions`.
 
 3.  Create a `zotero-plugin.ini` file as described in the
-    [zotero-plugin README](https://github.com/retorquere/zotero-plugin#readme).
+    [zotero-plugin][zotero-plugin] README.
     This file is used by the [`zotero-start`][zotero-start] command to determine
     where to install the extension when running a development build.
 
-4.  Install dependencies:
+4.  In order to install the forked `zotero-plugin` package, you must
+    [log in to the GitHub Packages npm registry][github-npm-auth]:
+
+        npm login --scope=@dvanoni --registry=https://npm.pkg.github.com
```

Comment:
This results in the following for me: 

```
❯ npm login --scope=@dvanoni --registry=https://npm.pkg.github.com
Username: balopat
Password: 
Email: (this IS public) patobalint@gmail.com
npm ERR! code E403
npm ERR! 403 403 Forbidden - PUT https://npm.pkg.github.com/-/user/org.couchdb.user:balopat - Permission denied
npm ERR! 403 In most cases, you or one of your dependencies are requesting
npm ERR! 403 a package version that is forbidden by your security policy.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/balopat/.npm/_logs/2022-12-24T01_47_35_258Z-debug.log
```

Reply:
@balopat, for the password field, did you enter a personal access token with the `read:packages` scope?

Reply:
Hmmm...nope, I just used my password for npm...
But I generated now a read-only token and still doesn't work, still `npm ERR! 403 403 Forbidden - PUT https://npm.pkg.github.com/-/user/org.couchdb.user:balopat - Permission denied`

Reply:
Hmm, that's strange. I just tried the process again myself and was able to successfully log in.

Just to confirm that we're trying to use the same type of token, do you want to try creating one using this link?
https://github.com/settings/tokens/new?scopes=read:packages

Reply:
Aaahhh, I totally misunderstood what's happening here and tried to use an npmjs.com token instead of a github token...okay, that works now :) let me try from the start, a clean setup without the local installation.

Reply:
and I confirm that `npm ci` works like a charm.